kernel/object-store/src/main/java/io/mosip/commons/khazana/impl/S3Adapter.java

```diff
@@ -352,8 +350,10 @@ private AmazonS3 getConnection(String bucketName) {
         if (os != null && os.size() > 0) {
             List<ObjectDto> objectDtos = new ArrayList<>();
             os.forEach(o -> {
-                // ignore the Tag file
-                String[] tempKeys = o.getKey().endsWith(TAGS_FILENAME) ? null : o.getKey().split("/");
+				// ignore the Tag file
+				String[] tempKeys = o.getKey().split("/");
+				if (tempKeys[1] != null && tempKeys[1].endsWith(TAGS_FILENAME))

```

Comment:
this line will work for both useAccountAsBucketname as true and false ?

Reply:
Fix  : if (useAccountAsBucketname) {
					if (tempKeys[1] != null && tempKeys[1].endsWith(TAGS_FILENAME))
						tempKeys = null;
				} else {
					if (tempKeys[0] != null && tempKeys[0].endsWith(TAGS_FILENAME))
						tempKeys = null;
				}